routes.py

```diff
@@ -390,10 +397,37 @@ def createComment():
         status = "success"
     return flask.jsonify(status)
 
-
-@app.route("/search", methods=["POST"])
+@app.route("/likeAPost", methods = ["POST"])
+@login_required
+def likeAPost():
+    postId = 
```

Comment:
Just wanna double check, when working on the front end for this later this line will become:
`postId = flask.request.args.get("postId")` ?

Reply:
Yes it will! I've updated it on my end but haven't committed it yet but it will have that and then the line other needs the author ID sent through